configure.ac

```diff
@@ -213,6 +238,7 @@ AM_CONDITIONAL([netmap_support], [test x$netmap_support = xyes ])
 AM_CONDITIONAL([PKTIO_DPDK], [test x$pktio_dpdk_support = xyes ])
 AM_CONDITIONAL([HAVE_PCAP], [test $have_pcap = yes])
 AM_CONDITIONAL([SDK_INSTALL_PATH_], [test "x${SDK_INSTALL_PATH_}" = "x1"])
+AM_CONDITIONAL([DPDK_DEFAULT_DIR], [test "x${DPDK_DEFAULT_DIR}" = "x1"])
```

Comment:
I'd suggest to move AC_ARG_WITH([dpdk-path], ...) to separate file, AC_DEFUN it to some macro name
and then AC_REQUIRE it from linux-generic's odp_dpdk.m4 and from linux-dpdk's configure.m4. This way we would have a consistent way of specifying DPDK location.